cmd/nerdctl/compose_config_test.go

```diff
@@ -0,0 +1,50 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/testutil"
+)
+
+func TestConfig(t *testing.T) {
+	base := testutil.NewBase(t)
+
+	var dockerComposeYAML = `
+version: '3.1'
+
+services:
+  hello:
+    image: docker.io/nginx
```

Comment:
Invalid image name

Reply:
@afbjorklund  list of image that you may use https://github.com/containerd/nerdctl/blob/1f817eb920470c491ba350c10f6df4eacaf7c0cd/pkg/testutil/testutil_linux.go#L26

Reply:
Done